src/errors.ts

```diff
@@ -54,6 +54,20 @@ export interface ForbiddenErrorAPIResponse {
 	message: string;
 }
 
+/**
+ * The response returned by the Prismic Custom Types API bulk transaction
+ * endpoint when a soft or hard limit is reached.
+ */
+type BulkTransactionAPIResponse = {
+	details: {
+		customTypes: {
```

Comment:
I could be wrong, but I didn't see anything about slices in Slice Machine's `/bulk` error handling; SM only validates that the `customTypes` object is present.

I think this is because limitation errors should only happen if too many documents need to be deleted. Documents are only deleted if custom types are delete, which does not concern slices.